notes/ltc-cola-summary.vbs

```diff
@@ -301,7 +305,7 @@ If approval_summary_check = 1 then
     EndDialog
 
     Do
-        Dialog Dialog1 
+        Dialog Dialog1
```

Comment:
worker_signature is in dialog, but not mandatory in the do....loop. Either remove variable from dialog or add handling and error messaging.